content/en/docs/languages/python/propagation.md

```diff
@@ -9,12 +9,30 @@ Although not limited to tracing, it is what allows traces to build causal
 information about a system across services that are arbitrarily distributed
 across process and network boundaries.
 
-## Manual W3C Trace Context Propagation
+OpenTelemetry provides a text-based approach to propagate context to remote
+services using the [W3C Trace Context](https://www.w3.org/TR/trace-context/)
+HTTP headers.
 
-The following generic example demonstrates how you can propagate trace context
+## Automatic context propagation
+
+Auto-instrumentation exists for some popular Python frameworks and libraries,
+such as Jinja2, Flask, Django, and Celery.
```

Comment:
nit: shall we add a link about this? which could make the reader easier to find reference. https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation#readme

Reply:
This could be helpful, but I think we should also include instructions on how to configure the propagators with environment variables for example using the `OTEL_PROPAGATORS`. Also, I do not see any reference to how to use the propagate API. Maybe it could be done separately in follow-up PR.